frontend/javascripts/oxalis/model/sagas/mesh_saga.ts

```diff
@@ -585,7 +626,11 @@ function* _refreshMeshWithMap(
     );
     shouldBeRemoved = false;
   }
-
+  // see comment in l. 292
```

Comment:
line numbers can get quickly out of sync. I'd extract the common code into a helper function and put the comment there.